pretender.js

```diff
@@ -154,11 +155,12 @@ function interceptor(pretender, nativeRequest) {
     }
 
     FakeXMLHttpRequest.prototype.send.apply(this, arguments);
-    if (!pretender.checkPassthrough(this)) {
-      pretender.handleRequest(this);
-    } else {
+
```

Comment:
This is the same, isn't it ?

Reply:
yes, I think its just better to test the truth and fallback in the else after, wdyt?